package.json

```diff
@@ -6,9 +6,19 @@
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "city": "^1.0.4",
```

Comment:
remove any dependency that you are not using

there something dependencies hell, try to read